core/controllers/base.py

```diff
@@ -165,32 +182,38 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             self.payload = None
         self.iframed = False
 
+        self.user_id = None
+        self.username = None
+        self.email = None
+        self.partially_logged_in = False
+        self.user_is_scheduled_for_deletion = False
+        self.current_user_is_super_admin = False
+
         try:
             auth_claims = auth_services.get_auth_claims_from_request(request)
         except auth_domain.StaleAuthSessionError:
-            logging.info(
```

Comment:
Why remove this log?

Reply:
I didn't find it valuable since I don't think it "matters" how often a session becomes stale.